pixi/src/checks/PageExperienceCheck.js

```diff
@@ -89,7 +112,7 @@ export default class PageExperienceCheck {
             fid: {
               id: 'fid',
               unit: UNIT_MS,
-              data: this.createLabData(labData['interactive']),
+              data: this.createLabData(labData['total-blocking-time']),
```

Comment:
Note also that the lab data copy should be identified as `Interactivity | TBT` not `FID`